src/assets/less/components/tags-input.less

```diff
@@ -67,7 +67,6 @@
 		box-shadow: 0 2px 2px rgba(0, 0, 0, .04);
 
 		@media (min-width: @screen-sm-min) {
-			overflow-x: unset;
```

Comment:
This was here for a reason.

![image](https://github.com/user-attachments/assets/c58b5fb9-165a-407e-8a5f-f89000dc8cf1)

Reply:
Hm, true, but it breaks styles in other way. I'll check it again.

Reply:
Should be OK now